cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        createPost(argv)
+    }
+})
+//update post command
+y.scriptName("connectme")
+y.command({
+    command: 'update-post',
+    describe: 'updates a post',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        updatePost(argv)
+    }
+})
+
+// Delete post
+y.scriptName("connectme")
+y.command({
+    command: 'delete-post',
+    describe: 'deletes a post',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Not possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Not possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+
+    },
+    handler(argv) {
+        deletePost(argv)
+    }
+})
+
+// View comment
+y.scriptName("connectme")
+y.command({
+    command: 'view-comment',
+    describe: 'view a comment',
+    builder: {
+        post: {
+            describe: 'Post name',
+            demandOption: true,
+            type: 'string'
+        },
+        canLike: {
+            describe: 'Possibility to like',
+            demandOption: true,
+            type: 'bool'
+        },
+       canUnlike: {
+           describe: 'Possibility to unlike',
+            demandOption: true,
+            type: 'bool'
+        },
+        likes: {
+            describe: 'Ammount of likes',
+            demandOption: true,
+            type: 'int'
+        },
+        tag: {
+            describe: 'Tag',
+            demandOption: true,
+            type: 'string'
+        },
+
+    },
+    handler(argv) {

```

Comment:
it looks like the comment is the same as the post in here

Reply:
fixed